test/gesture_detectors_test.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_hooks/src/framework.dart';
+import 'package:flutter_hooks/src/hooks.dart';
+
+import 'mock.dart';
+
+void _testGestureRecognizer<T>({
+  Type gestureRecognizer,
+  T Function() useHook,
+}) {
+  group('use$gestureRecognizer', () {
```

Comment:
Could we add a test where the gesture detector is used, with a click & all?

Reply:
Done!